deleteandearn.java

```diff
@@ -0,0 +1,31 @@
+//Time Complexity: O(NlogN)
```

Comment:
We are not sorting or performing any other operations of complexity NlogN. It would be O(N) as we are iterating through the array